content/realtime/channels/channel-parameters/rewind.textile

```diff
@@ -0,0 +1,87 @@
+---
+title: Rewind
+section: realtime
+index: 2
+jump_to:
+  Help with:
+    - Overview#overview
+    - Examples#examples
+---
+
+h2(#overview). Overview
+
+The @rewind@ parameter allows you to specify, at the time of attaching to a channel, where to start the attachment from. 
```

Comment:
I think like the other feedback provided, this is too much detail for someone arriving.

Story is:

* Rewind allows you to attach to a channel and automatically replay messages previously published on a channel. This is useful because of X/Y
* Here is a code sample.
* Here is more detailed info

Reply:
Is the issue here the fact that there is too much information on the page, or that you think the code samples should come before the more detailed information?

Reply:
I think the issue is as follows:

1. Rewind is conceptually simple, yet the reader has to read through 25 lines of copy to understand it.
2. All the technical details behind how rewind edge cases work, should be out of the readers way when they arrive.  They want to know - what is rewind, why is it useful, how can I use it, show me the code, OK, now what else should I know now that I understand it.  This last bit is where I think a lot of this documentation should live.

Reply:
I've briefly addressed this with moving some of the copy around so it's not so front loaded. It's something we should look at separately across all the docs to make sure we're consistent in future.